### DISCUSSION TITLE
How can I customize the app using a `styles.css` file?

### DISCUSSION THREAD

**Discussion by GitHunter0** [flet-dev/flet#3906] [Q&A]
### Question

Hi, I want to further customize a flet app with css but I found nothing about on the docs and on the internet. Thank you

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by tanmay-bhatgare** [comment]
I don't think such a simple thing needed an explanation. Since Flet is a Python framework built on top of Flutter, and Flutter doesn't support custom styling beyond the default options, neither does Flet.

**Comment by GitHunter0** [comment]
@tanmay-bhatgare , what is trivial to you might not be for other people. I'm new to Flutter, one line in the documentation would be enough to clarify my question.

**Reply by tanmay-bhatgare** [reply]
@GitHunter0 You might not be familiar with Flutter, but you should do some research before raising an issue because the maintainer of this repo has other things to do than to solve a simple issue which can be solved using a Google search.

**I’m not saying that you did anything wrong by asking this question, but you shouldn’t raise such simple issues. Consider searching on Google first. Happy hacking.**
